examples/auto-resume-sandbox/manifests/envoy-gateway/tenant-a-infra.yaml

```
@@ -0,0 +1,149 @@
+metadata:
+  name: sandbox-gateway-svc
+  namespace: tenant-a
+spec:
+  selector:
+    app: sandbox-gateway
+  ports:
+  - name: grpc-ext-proc
+    port: 50051
+    targetPort: 50051
+# 7. BackendTrafficPolicy setting 180s HTTP request timeout for tenant-a cold-starts
+apiVersion: gateway.envoyproxy.io/v1alpha1
```

YAMLlint (1.37.1)

[error] duplication of key "apiVersion" in mapping

(key-duplicates)